app/src/main/java/com/example/gradeassist/ui/home/HomeFragment.kt

```diff
@@ -26,6 +28,19 @@ class HomeFragment : Fragment() {
         homeViewModel.text.observe(this, Observer {
             textView.text = it
         })
-        return root
+
+        val v = inflater.inflate(R.layout.fragment_home, container, false)
+        val button = v.findViewById<View>(R.id.button) as Button
+
+        button.setOnClickListener()
+        {
+            val fragment = QuizResultsFragment()
+            val fragmentManager = activity!!.supportFragmentManager
+            val fragmentTransaction = fragmentManager.beginTransaction()
+            fragmentTransaction.replace(R.id.container, fragment)
+            fragmentTransaction.addToBackStack(null)
+            fragmentTransaction.commit()
+        }
+        return v
```

Comment:
made button functional now the quiz 1 results button redirects you to the quiz results fragment page (which was blank).